src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,161 @@
+package lotto.controller;
+
+import lotto.domain.*;
+import lotto.domain.Dto.RankCountDto;
+import lotto.service.LottoService;
+import lotto.utils.SplitUtil;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoController {
+    private final LottoService lottoService;
+    private final InputView inputView;
+    private final ResultView resultView;
+
+    public LottoController(InputView inputView, ResultView resultView, LottoService lottoService) {
+        this.inputView = inputView;
+        this.resultView = resultView;
+        this.lottoService = lottoService;
+    }
+
+    public void run() {
+        PurchaseAmount purchaseAmount = createPurchaseAmount();
+        LottoTickets lottoTickets = createLottoTickets(purchaseAmount);
+        printLottoTickets(lottoTickets);
+
+        WinningNumbers winningNumbers = createWinningNumbers();
+        BonusBall bonusBall = createBonusBall(winningNumbers);
+        RanksCount ranksCount = createRanksCount(winningNumbers, lottoTickets);
+        matchWith(ranksCount, bonusBall);
+        printStatistics(createRanksCountDtos(ranksCount));
+
+        ProfitRate profitRate = createProfitRate(ranksCount, purchaseAmount);
+        printProfitRate(profitRate);
+    }
+
+    protected PurchaseAmount createPurchaseAmount() {
+        try {
+            String purchaseAmount = inputView.purchaseAmount();
+            return new PurchaseAmount(purchaseAmount);
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+            return createPurchaseAmount();
+        }
+    }
+
+    protected LottoTickets createLottoTickets(PurchaseAmount purchaseAmount) {
+        LottoTicketPrice lottoTicketPrice = new LottoTicketPrice();
+        TicketOffice ticketOffice = new TicketOffice(lottoTicketPrice);
+
+        TotalNumberOfTicket totalNumberOfTicket = new TotalNumberOfTicket(purchaseAmount, lottoTicketPrice);
+        ManualNumberOfTicket manualNumberOfTicket = manualNumberOfTicket(totalNumberOfTicket);
+        AutoNumberOfTicket autoNumberOfTicket = new AutoNumberOfTicket(manualNumberOfTicket, totalNumberOfTicket);
+
+        LottoTickets manualLottoTickets = createManualLottoTickets(manualNumberOfTicket);
+        LottoTickets autoLottoTickets = createAutoLottoTickets(totalNumberOfTicket, manualNumberOfTicket);
+        resultView.purchaseTickets(manualNumberOfTicket.numberOfTicket(), autoNumberOfTicket.numberOfTicket());
+
+        return ticketOffice.totalTickets(manualLottoTickets, autoLottoTickets);
```

Comment:
```suggestion
        return new LottoTickets(manualLottoTickets, autoLottoTickets);
```
지금은 `ticketOffice` 가 하는일이 없어 보이네요.

Reply:
말씀하신대로 `TicketOffice`가 제대로 활용이 되지 않고 있었네요... 해당 엔티티 제거 후 
`LottoTickets`에서 수동, 자동 티켓들 합치는 로직 추가하여 적용하였습니다